crates/noirc_frontend/src/hir/def_collector/dc_mod.rs

```diff
@@ -287,6 +287,7 @@ impl<'a> ModCollector<'a> {
         }
 
         for item in &trait_def.items {
+            // TODO(Maddiaa): Should traits have a notion of attributes? -> I have a feeling they should
```

Comment:
Can you create an issue for this?

Reply:
I could implement it now if you'd perfer, i am just not sure if it is by design